Orange/data/domain.py

```diff
@@ -58,35 +55,52 @@ def __init__(self, source, destination):
 
 
 def filter_visible(feats):
-    """
-    Args:
-        feats (iterable): Features to be filtered.
+    """Filter only visible (not hidden) features.
+
+    Parameters
+    ----------
+    feats : Iterable
+        Features to be filtered.
 
-    Returns: A filtered tuple of features that are visible (i.e. not hidden).
+    Returns
+    -------
+    tuple
+        A filtered tuple of features that are visible (i.e. not hidden).
     """
     return (f for f in feats if not f.attributes.get('hidden', False))
```

Comment:
My bad 🙈 As far as I remember this was meant to be a generator (and is just documented wrongly). Feel free to fix 😉